README.md

```diff
@@ -1 +1,84 @@
+
+![menu](./diagrams/projects.svg)
+
+#### Project Details
+
+The project details page shows your balance, your unique JSON-RPC url that you should use that contains your `Project ID` to authenticate the API calls, your registered addresses, your transactions history and account history.
+
+###### Balance
+
+This section is where you can add your credit card with FuelStop, and after which displays information about the account associated with the project. The Mainnet project would start without credit card added, and therefore you have to add one before we can fund your transactions. The Ropsten project, however, is with a testnet of Ethereum, and does not require you to add any credit card to start using. It comes with a fake credit card, and you can top it up anytime you want by clicking on the "Top Up Button" with no cost. This is where you can test your system or library with FuelStop.
+
+![Balance without credit card](./diagrams/balance-without-credit-card.svg)
+
+![FuelStop Documentation](./diagrams/balance.svg)
+
+###### Details
+
+Here you can find the name of the network the project is associated with, and your RPC URL. FuelStop's Ethereum JSON-RPC APIs require a valid project ID to be part of your URL to autheticate the calls. This is how we identify who you are behind the API requests, and it is important to keep it as secret to yourself.
+
+![Details](./diagrams/details.svg)
+
+###### Addresses
+
+Before sending your signed transactions to us via `eth_sendRawTransaction`, you have to register your address with us. If a signed transaction that is not from any of the registered addresses gets sent to FuelStop via your RPC URL, FuelStop will behave no differently than regular Ethereum nodes and simply broadcase it the network. If your address has some Ether but not enough to cover the particular transaction's fees, FuelStop would only fund your address to the  exact amount that is enough for the transaction to go through.
+
+![Addresses](./diagrams/addresses.svg)
+
+###### Transactions
+
+Transactions table contains your transaction history. You can keep refreshing to monitor the status of your transactions. Your transaction status could be as follows.
+
+1. `New`: The transaction has been received and is waiting for the fund transaction to be mined.
+2. `Sent`: The transaction has been broadcast to the network and is waiting for it to be mined.
+3. `Confirmed`: The transaction has been mined in a block and transaction execution is successful on-chain.
+4. `Failed`: The transaction has been mined in a block and transaction execution is failed on-chain. This is likely because you sent your transaction to a smart contract, and part of the contract logic decided to reject any state transition.
```

Comment:
```suggestion
4. `Failed`: The transaction has been mined in a block and transaction execution has failed on-chain. This is likely because you sent your transaction to a smart contract, and part of the contract logic has rejected a state transition.
```